common/Strings/en-us/Resources.resw

```diff
@@ -205,12 +205,31 @@
     <value>Uptime:</value>
     <comment>Title for the uptime property of a compute system. (The amount of time its been running)</comment>
   </data>
+  <data name="HyperVAdminAddUser" xml:space="preserve">
+    <value>Add User to Hyper-V admin group</value>
+  </data>
+  <data name="HyperVErrorTitle" xml:space="preserve">
+    <value>Microsoft Hyper-V</value>
+    <comment>Title text for the UI showing errors for user not part of Hyper-V admin group</comment>
+  </data>
+  <data name="RestartButton" xml:space="preserve">
+    <value>Restart Now</value>
+    <comment>Text for button that restarts the system.</comment>
+  </data>
+  <data name="RestartMessage" xml:space="preserve">
+    <value>You must restart the computer after adding the user to the Hyper-V admin group for changes to take effect.</value>

```

Comment:
Are these strings approved by our content peeps?

Reply:
If they're not reviewed/approved yet and won't be for this release, please make a Task issue for changing the strings and put it in the May 0.14 milestone.

Reply:
Got it. Thanks.